### PR TITLE
Count wandering traders as NPCs for /butcher.

### LINKED ISSUE #1347: Wandering Villagers are including when butchering
**Versions**

WorldEdit version: 7.1.0;8e55131
Platform version: Paper 1.15.2 b304

**Describe the bug**

  When using the command "/butcher", **Wandering Villagers** are also removed alongside the "mobs"

**To Reproduce**

1. use /butcher

**Expected behavior**

  Wandering Villagers are considered a villager or at least a passive mob.

### PR DESCRIPTION
**Pull request by wizjany**
Fixes #1347.

### CODE REVIEW

File: worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitEntityProperties.java [hunk 1]
```diff
@@ -46,6 +47,18 @@
 
 class BukkitEntityProperties implements EntityProperties {
 
+    private static final boolean hasAbstractVillager;
```
Comment (octylFractal) [inline worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitEntityProperties.java:50]: `HAS_ABSTRACT_VILLAGER`?

File: worldedit-bukkit/src/test/java/com/sk89q/wepif/TestOfflinePermissible.java [hunk 2]
```diff
@@ -185,4 +188,94 @@ public Location getBedSpawnLocation() {
     public Map<String, Object> serialize() {
         throw new UnsupportedOperationException("Not supported yet.");
     }
+
```
Comment (octylFractal) [inline worldedit-bukkit/src/test/java/com/sk89q/wepif/TestOfflinePermissible.java:191]: Curious if this might be easier to do with a mock or a Proxy.

Comment (wizjany) [reply worldedit-bukkit/src/test/java/com/sk89q/wepif/TestOfflinePermissible.java:191]: maybe in the long run.

File: worldedit-bukkit/build.gradle.kts [hunk 3]
```diff
@@ -22,10 +22,10 @@ configurations.all {
 dependencies {
     "api"(project(":worldedit-core"))
     "api"(project(":worldedit-libs:bukkit"))
-    "api"("org.bukkit:bukkit:1.13.2-R0.1-SNAPSHOT") {
+    "api"("org.bukkit:bukkit:1.15.2-R0.1-SNAPSHOT") {
```
Comment (me4502) [inline worldedit-bukkit/build.gradle.kts:25]: I wonder if there's something we can do to ensure that we don't use APIs that will break 1.13.2 compat

**Pull request merged**